src/components/TimeCardPage/TimeSheet.tsx

```diff
@@ -61,21 +122,49 @@ export default function Page() {
         setCurrentTimesheetsToDisplay (userTimesheets, date); 
     }
     
-    const [userTimesheets, setUserTimesheets] = useState([]); 
-    const [currentTimesheets, setCurrentTimesheets] = useState([]);
-    const [selectedTimesheet, setTimesheet] = useState();
+    // fetch the information of the user whos timesheet is being displayed
+    // if user is an employee selected and user would be the same
+    // if user is a supervisor/admin then selected would contain the information of the user
+    // whos timesheet is being looked at and user would contain the supervisor/admins information
+    // by default the first user is selected
+    const [selectedUser, setSelectedUser] = useState<UserSchema>();
+    const [user, setUser] = useState<UserSchema>();
+
+    // associates is only used by supervisor/admin for the list of all associates they have access to
+    const [associates, setAssociates] = useState<UserSchema[]>();
+
+    const [userTimesheets, setUserTimesheets] = useState<TimeSheetSchema[]>([]); 
+    const [currentTimesheets, setCurrentTimesheets] = useState<TimeSheetSchema[]>([]);
+    const [selectedTimesheet, setTimesheet] = useState<TimeSheetSchema>();
     const columns = defaultColumns 
 
-    //Pulls user timesheets, marking first returned as the active one
+    // this hook should always run first
+    useEffect(() => {
+        apiClient.getUser().then(userInfo => {
```

Comment:
Doesn't necessarily need to be here, but consider documenting somewhere (docs, comments, github pages, etc.) why we are hitting the backend to get the user information rather than just using the amplify `currentAuthenticatedUser` method.

From what we talked about, I think the major reason is that there are attributes we are going to need for front-end display purposes like their company that will come from Dynamo, rather than Cognito attributes. It's something that might need to be revisited later (i.e. will we actually need to display their company anywhere? Or will it be enough to just get the user timesheets, which is auto-sorted by company on the backend), so will be good to document our decision rn.